### PR TITLE
Remove commented fields from create command outputs

### PR DESCRIPTION
**Pull request by Orzelius**
with this addition I'm undoing my very first contribution to garden (as an employee). #2904

* create commands no longer add empty commented fields
* remove `--skip-comments` flag
* update getting-started docs accordingly
* add links do docs on top of generated docs

image

### CODE REVIEW

File: core/src/commands/create/create-project.ts [hunk 1]
```diff
@@ -155,6 +154,9 @@ export class CreateProjectCommand extends Command<CreateProjectArgs, CreateProje
       },
     })
 
+    const projectDocURL = "https://docs.garden.io/using-garden/projects"
+    yaml = `# More info and documentation at ${projectDocURL}\n\n${yaml}`
```
Comment (edvald) [inline core/src/commands/create/create-project.ts:158]: Maybe add a link to the reference docs as well.

File: core/src/commands/create/create-module.ts [hunk 2]
```diff
@@ -207,6 +206,8 @@ export class CreateModuleCommand extends Command<CreateModuleArgs, CreateModuleO
       },
     })
 
+    const moduleTypeUrl = getModuleTypeUrl(type)
+    yaml = `# Documentation about ${type} modules at ${moduleTypeUrl}\n\n${yaml}`
```
Comment (edvald) [inline core/src/commands/create/create-module.ts:210]:
```suggestion
    yaml = `# See the documentation and reference for ${type} modules at ${moduleTypeUrl}\n\n${yaml}`
```

File: core/src/commands/create/create-project.ts [hunk 3]
```diff
@@ -189,16 +191,13 @@ export class CreateProjectCommand extends Command<CreateProjectArgs, CreateProje
     log.info("")
 
     // This is to avoid `prettier` messing with the string formatting...
-    const formattedIgnoreName = chalk.bold.white(".gardenignore")
```
Comment (edvald) [inline core/src/commands/create/create-project.ts:192]: I still think mentioning the .gardenignore file is good.

**Review by edvald — CHANGES_REQUESTED** [review]
Some minor comments, otherwise 👍

File: core/src/commands/create/create-project.ts (hunk 3 above)
Comment (Orzelius) [reply core/src/commands/create/create-project.ts:192]: It is still mentioned. You can see from the first few lines in the picture in the pr

File: core/src/commands/create/create-project.ts (hunk 1 above)
Comment (Orzelius) [reply core/src/commands/create/create-project.ts]: sure, why not. Added in [59c0199](https://github.com/garden-io/garden/pull/3068/commits/59c0199a4f5d31df088f3e1c1905176176c449c4)

**Review by vvagaytsev — APPROVED** [review]
LGTM!

**Review by edvald — APPROVED** [review]

**Pull request merged**